README.md

```diff
@@ -0,0 +1,167 @@
+
+First, notice that the method passed into `get` (written with the ES6 "fat arrow" syntax) takes two arguments: `request` and `response`. These are very important objects and you will be using them a lot when you work with Node. For now, what you should know is that the `request` object contains information as well as method relating to the "request" that has come in from the client. The `response` object, as you may have guessed, contains information and methods about how the server's response.
+
+Moving on, do you notice anything familiar about this function that we've passed into the `get` method? Do you recognize this pattern?
+
+That's right. It's a callback! The asynchronous character of the process may not have been immediately obvious to you, but it's the same idea that was explained in the previous lesson: The server is waiting for a GET request to the path `/`. When will that request come in? No one knows. It will happen at some indeterminate point in the future. And when it does, the `get` method will call the callback function that we supply.
+
+Bottom line here: Congratulations, you're about to write some asynchronous JS code with callbacks!
+
+You may remember from our lesson on callbacks that one way to explain a callback function is to say that it encapsulates the logic about how our program should continue in the future. Keeping this in mind how should our program proceed once a get request to `/` is received? At the moment, the only code inside the callback function is a comment, directed at you dear learner, and a call to a method `end` on the `response` object. This is an important function. Let's see if you can modify the code inside in this callback to get the test for this route in our testing file (`test/server_test.js`) to pass.
+
+> In order to get oriented here, it might be a good idea to take a look at the node documentation for the `end` method on the `response` object. Also, if you look at the test, you'll notice that the test checks to see if the `Content-Type` "Headers", which signal to the browser what kind of data has been set appropriately. In this case, we are just sending some text so we can use the type "plain/text".
+
+When testing your code, you can of course run `npm test` to check if your code is passing the Hello World test passes. However, if you would like to do some manual testing, you can start the server by opening your terminal and running `node run-server.js`. This will start the server on port 3000. You can then test the server in your browser, by going to `http://localhost:3000`, or, better yet, test it by calling the browser using the curl command line tool, like so:
+
+```bash
+curl -X GET http://localhost:3000
+```
+
+> Note: We will be using the curl command line tool in the next section of this lab, so you may want to get familiar with it.
```

Comment:
I'm not positive, but I suspect that students running the lab on Nitrious.io will have trouble with cURL. I think we should definitely keep this bit on curl in here, but just as a heads-up.